H2TauTau/scripts/multiHarvest.py

```diff
@@ -26,13 +23,13 @@ def get_options():
                       help='Version of CMSSW used to produce the samples.')
     parser.add_option("-l", "--prod_label", dest = "prod_label",
                       default='diTau_2018_modular_cfg',
-                      help='Heppy cfg file used to produce the samples.')
-    parser.add_option("-g", "--grep", dest = "cut_on_sample_names",
+                      help='The prod label or part of prod labels to be selected.')
+    parser.add_option("-C", "--component", dest = "cut_on_sample_names",
```

Comment:
Ok for the way it is used now. It used to pass `cut_on_sample_names` to the `grep` command on paths so it could also have been used to select more generally directories to harvest.

Reply:
yes but previously the label had to be perfectly right, meaning you could not harvest different systematics of the same production.